x-pack/plugins/synthetics/server/synthetics_service/get_service_locations.ts

```diff
@@ -45,12 +45,7 @@ export async function getServiceLocations(server: SyntheticsServerSetup) {
       locations: Record<string, ManifestLocation>;
     }>(server.config.service!.manifestUrl!);
 
-    const availableLocations =
-      server.isDev || server.config.service?.showExperimentalLocations
-        ? Object.entries(data.locations)
-        : Object.entries(data.locations).filter(([_, location]) => {
-            return location.status === LocationStatus.GA;
-          });
+    const availableLocations = Object.entries(data.locations);
```

Comment:
As you are removing the `showExperimentalLocations` should it be the `false` statement from the old code?
```
Object.entries(data.locations).filter(([_, location]) => {
            return location.status === LocationStatus.GA;
          });
```

Reply:
So, essentially what this change does is switch the "dev" behavior to _all_ behavior.

Synthetics Service locations have a `status` field that can be `'GA' | 'beta' | 'etc'`, and anything _not_ `GA` is treated as experimental. The thing is, we don't actually utilize this feature in the Service release flow right now, and until we do we will remove this notion from Kibana.

There's a `showExperimentalLocations` flag that must be enabled for QA and Staging Kibana to pick up the locations we have for those regions. With this removed, we won't need to set that flag. This will simplify the cross-env config we need to set up in the Kibana Controller.

Thus, the outcome we truly want from this change is to return all locations, regardless of their `status`.